.github/workflows/release-service-stream.yml

```diff
@@ -0,0 +1,90 @@
+name: 'Release: Stream Service'
+
+on:
+  push:
+    tags:
+      - 'stream-v**'
+
+env:
+  # DockerHub login
+  DOCKERHUB_USER: "keboolabot"
+  DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  IMAGE_TAG: ${{ github.ref_name }}
+  IMAGE_NAME: "keboola/stream-service"
+ 
+  ECR_REGION: "us-east-1"
+  ECR_REPOSITORY: "keboola/stream-service"
+  ECR_PUSH_ROLE: "arn:aws:iam::968984773589:role/kbc-ecr-StreamServicePushRole-k57lBKJxl9Dr"
+  ACR_REGISTRY: "keboola.azurecr.io"
+  ACR_REPOSITORY: "stream-service"
+  ACR_USERNAME: "stream-service-push"
+  GCP_REGISTRY: "us-central1-docker.pkg.dev"
+  GCP_IDENTITY_PROVIDER: "projects/388088979044/locations/global/workloadIdentityPools/github/providers/github"
+  GCP_ACCOUNT: "keboola-as-code-ci-push@keboola-prod-artifacts.iam.gserviceaccount.com"
+  GCP_REPOSITORY: "keboola-prod-artifacts/keboola-as-code/stream-service"
+
+jobs:
+  test-lint:
+    name: "Lint"
+    secrets: inherit
+    uses: ./.github/workflows/test-lint.yml
+  test-unit:
+    name: "Unit Tests"
+    secrets: inherit
+    uses: ./.github/workflows/test-unit.yml
+  test-e2e-service-stream:
+    name: "E2E: Stream"
+    secrets: inherit
+    uses: ./.github/workflows/test-e2e-service-stream.yml
+#  test-k8s-service-stream:
+#    name: "K8S: Stream"
+#    secrets: inherit
+#    uses: ./.github/workflows/test-k8s-service-stream.yml
```

Comment:
Why is this commented? Due to possible release?

Reply:
Yes, release and related tests are WIP.